frontend/packages/model-registry/manifests/base/model-registry-ui-deployment.yaml

```
@@ -0,0 +1,65 @@
+  labels:
+    app: model-registry-ui
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: model-registry-ui
+  template:
+    metadata:
+      labels:
+        app: model-registry-ui
+    spec: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)